index.html

```diff
@@ -80,5 +80,192 @@ <h2 id="h2"><p>I'm a Software Developer.</p></h2>
 
     <base1 id="base1"></base1>
 
+
+    <recentwork id="recentwork"><p id="recentwork_text">My Recent Works</p>
+    <img src="./assets/Vector.svg" alt="">
+
+    <frame44 id="frame44">
+        <featured_img id="featured_img">
+            <img_placeholder>
+            <img src="./assets/Img Placeholder.svg" alt=""></img_placeholder>
+        </featured_img>
+
+        <right_block id="right_block">
+            <p id="RB_title_post">Multi-Post Stories</p>
+            <p id="RB_secondary_text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul id="RB_tags">
+                <li class="RB_tags"><p class="RB_tags_text">css</p></li>
+                <li class="RB_tags"><p class="RB_tags_text">html</p></li>
+                <li class="RB_tags"><p class="RB_tags_text">bootstrap</p></li>
+                <li class="RB_tags"><p class="RB_tags_text">ruby</p></li>
+            </ul>
+            <a href="https://github.com/glitcher255/micro-porfolio"><btn_enable id="btn_enable" class="orange_btns"><p id="btn_enable_text">See Project</p></btn_enable></a>
+          </right_block>
+
+        <ul id="frame43">
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
+            </action_btn>
+
+            <li id="card_data">
+              <card_title id="card_title"><p>Profesional Art Printing Data</p></card_title>
+              <card_secondary id="card_secondary"><p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p></card_secondary>
+              <ul id="card_tags">
+                <li class="card_btns"><card_btns_font class="card_btns_font">html</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">bootstrap</card_btns_font></li>
+                  <li class="card_btns"><card_btns_font class="card_btns_font">Ruby</card_btns_font></li>
+                  </ul>
+
+              </card_tags>
+            </li>
+          </card_works>
+
+          <card_works id="card_works">
+            <action id="action">
+              <img src="./assets/Mask Group.png" alt="">
+            </action>
+            <action_btn id="action_btn" class="orange_btns"><p id="action_btn_text">See Project</p>
```

Comment:
Kindly, I suggest you to use the "Button" tag, or "A" to very soon use it to display the details of your project.

Reply:
Grammer issues in this feedback, I assume you mean change action_btn to button? in which case I have done so.

Reply:
you need to put a link here, you can put a button or an "A" link

Reply:
Done.